src/esp/metadata/managers/AOAttributesManager.cpp

```diff
@@ -182,37 +185,49 @@ AOAttributesManager::preRegisterObjectFinalize(
   // ArticulatedObjectsAttributes must have a valid URDF_file it relates to or
   // it should not be registered.
   const std::string urdfFilePath = AOAttributesTemplate->getURDFPath();
+  const std::string urdfFullFilePath = AOAttributesTemplate->getURDFFullPath();
   if (urdfFilePath.empty()) {
     // URDF Filepath empty is bad
     ESP_ERROR(Mn::Debug::Flag::NoSpace)
         << "ArticulatedObjectAttributes template named `" << AOAttributesHandle
         << "` does not specify a valid URDF Filepath, so registration is "
            "aborted.";
     return core::managedContainers::ManagedObjectPreregistration::Failed;
-  } else if (!Cr::Utility::Path::exists(urdfFilePath)) {
+  } else if (!Cr::Utility::Path::exists(urdfFilePath) &&
+             !Cr::Utility::Path::exists(urdfFullFilePath)) {
```

Comment:
Do we need to check both? Shouldn't the full path be enough after construction?

Reply:
The full path might not exist yet, depending on what pathway the attributes took to get here.